protected/humhub/libs/Sort.php

```diff
@@ -10,11 +10,11 @@
 
 class Sort
 {
-    public static function sort(&$arr, $field = 'sortOrder')
+    public static function sort(&$arr, $field = 'sortOrder', $default = PHP_INT_MAX)
```

Comment:
Can you add some docs, describing the new default argument and how to use it. 

E.g. The default argument is used as default value for items without a given sort value. This might be a value lower than the actual max sort value in order to allow other items to be sorted after those default items.